sormas-api/src/main/java/de/symeda/sormas/api/sample/SampleSearchType.java

```diff
@@ -0,0 +1,13 @@
+package de.symeda.sormas.api.sample;
```

Comment:
Rename this to something like SampleAssociationType - SampleSearchType suggests a very specific use case, but we might also need this for something else in the future

Reply:
renamed